lib/core/src/sdk.rs

```diff
@@ -574,12 +574,9 @@ impl LiquidSdk {
             let mut current_liquid_block: u32 = 0;
             let mut current_bitcoin_block: u32 = 0;
             let mut shutdown_receiver = cloned.shutdown_receiver.clone();
-            let mut interval = tokio::time::interval(Duration::from_secs(10));
-            #[cfg(not(all(target_family = "wasm", target_os = "unknown")))]
-            interval.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Skip);
             loop {
                 tokio::select! {
-                    _ = interval.tick() => {
+                    _ = tokio::time::sleep(Duration::from_secs(10)) => {
```

Comment:
The previous interval implementation first tick was triggered immediately and here we are first waiting 10 seconds. Perhaps it is better to keep the current working behavior?

Reply:
Agreed. Done in [a64c3e7](https://github.com/breez/breez-sdk-liquid/pull/880/commits/a64c3e700dddb9fa105f48ec97b3363f81491d14) for the track new blocks task. IMO it isn't needed on the status stream ping.

I also adjusted the E2E tests to wait for the first sync before starting. The delayed first sync was causing the tests to fail. Now there is no delay any more but it's a good idea to wait for sync anyway.